CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+* (types) [#15433](https://github.com/cosmos/cosmos-sdk/pull/15433) Allow disabling of account address caches (for printing bech32 account addresses).
```

Comment:
The changelog should be under the bugfix or improvement section (whichever you think fits best).

Reply:
Moved it there. Am I finally doing it right? :)